components/scripts/network.settings

```diff
@@ -17,3 +17,7 @@
 
 # Uncomment to disable SSL certificate validation when connecting to the Gradle Enterprise server
 #ssl.allowUntrustedServer=true
+
+# Uncomment to use a custom connect or read timeout when fetching Build Scan data.
```

Comment:
Remove the dot at the end of the sentence

Reply:
✅ Done